File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/util/IterableUtil.java
package com.dkellycollins.triggerfinger.util;

import java.util.ArrayList;

public class IterableUtil {

    public static <T> ArrayList<T> toArrayList(Iterable<T> iterable) {
        ArrayList<T> arrayList = new ArrayList<T>();
        for(T item : iterable) {
            arrayList.add(item);
        }

        return arrayList;
    }

}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/util/IterableUtil.kt
package com.dkellycollins.triggerfinger.util

import java.util.ArrayList

object IterableUtil {

    fun <T> toArrayList(iterable: Iterable<T>): ArrayList<T> {
        val arrayList = ArrayList<T>()
        for (item in iterable) {
            arrayList.add(item)
        }

        return arrayList
    }

}
